rinja/src/lib.rs

```diff
@@ -170,9 +170,11 @@ impl<T: Template + ?Sized> Template for &T {
     const MIME_TYPE: &'static str = T::MIME_TYPE;
 }
 
-/// Object-safe wrapper trait around [`Template`] implementers
+/// [`dyn`-compatible] wrapper trait around [`Template`] implementers
```

Comment:
```suggestion
/// [`dyn`-compatible] wrapper trait around [`Template`] implementors.
```

In rust docs it's "implementors" so I suppose it's the correct word. ^^'

Reply:
Oops, thank you for proofreading!

(I'm like 99% sure that in English you are supposed to role a dice when to write -ter and when to write -tor. :D)

Reply:
Yup, and let's not even try to speak english, the pronunciation is even more random. 😆